gcloud/storage/bucket.py

```diff
@@ -167,9 +167,12 @@ def create(self, client=None):
         """
         client = self._require_client(client)
         query_params = {'project': client.project}
+        properties = dict(
+            (key, self._properties[key]) for key in self._changes)
+        properties['name'] = self.name
```

Comment:
What is `name` was already in there? Or is it impossible to add `name` to `_changes`?